packages/shared/friends/sagas.ts

```diff
@@ -708,6 +715,78 @@ export async function getFriendRequests(request: GetFriendRequestsPayload) {
   getUnityInstance().AddFriendRequests(addFriendRequestsPayload)
 }
 
+// New friend request flow
+export async function getFriendRequestsNew(request: GetFriendRequestsPayload) {
```

Comment:
Let's rename this to `getFriendRequestsProtocol` or `getFriendRequestsV2` FTTB, as new will shortly not be so.